app/views/layouts/admin/layout.html.haml

```diff
@@ -2,14 +2,15 @@
 %html
   %head
     %meta{charset: "utf-8"}
-    %title TfgTemplate: Admin
+    %title= page_title
```

Comment:
I will need to have a look at this admin/layout again, because it shares a lot with the main one / probably can get rid of some stuff in here because #82 kinda went past me unnoticed :|